### ISSUE TITLE
Move file on disk Option

### ISSUE THREAD

**Issue by rrrkey** [pornganizer/PGV4#97]
I need an option to move the file on disk, also renaming it, so I can properly **organize** my library.
I really see this as a must have feature.
It was available in V3 but it is gone in V4.

**Comment by pornganizer** [comment]
there is a "relocate" option in the right click menu on scene details
you can move the file, then relocate it to fix the path

**Comment by rrrkey** [comment]
Yes, this workaround is great, but after I organize in PG I have to organize on disk and the relocate each file,   one of the great things about v4 is the bulk edit, if i could move after that that would be perfect
